src/ralph/lib/mixins/fields.py

```diff
@@ -5,13 +5,13 @@
 from django.conf import settings
 from django.contrib.admin.widgets import AdminTextInputWidget
 from django.contrib.contenttypes.models import ContentType
-from django.core.exceptions import ValidationError
 from django.db import models
 from django.db.models.loading import get_model
 from django.forms.utils import flatatt
 from django.utils import six
 from django.utils.html import format_html, smart_urlquote
 from django.utils.translation import ugettext_lazy as _
+from rest_framework.exceptions import ValidationError
```

Comment:
if you swap exceptions like this, I'm afraid it'd not work in GUI (DRF's ValidationError is not descendant of Django's ValidationError)